server/routes/board.js

```diff
@@ -4,8 +4,23 @@ const express = require('express');
 
 const router = express.Router();
 
+const books = []; // use bookshelf to grab card data from database
+
 router.get('/', (req, res) => {
-  // show kanban board
+  console.log('fucking hitty');
```

Comment:
Nope. 🔥

Reply:
(Remember this is a public record of your work and professionalism), offline is fine, in code can be used against you